.circleci/config.yml

```diff
@@ -457,12 +562,12 @@ jobs:
               export CIRCLE_BUILD_NUM=300
             fi
             echo "export DOCKER_BRANCH_TAG=$(./scripts/docker_tag_from_branch.sh $CIRCLE_BRANCH)" >>$BASH_ENV
+            echo "export SHORT_GIT_HASH=$(echo $CIRCLE_SHA1 | cut -c -7)" >> $BASH_ENV
             echo "export VERSION=$(git rev-parse --short HEAD)" >> $BASH_ENV
             echo "export START_TIME=$(date +%Y%m%d-%Hh%Mm%Ss)" >>$BASH_ENV
             echo "export GCP_IMAGE=gcr.io/${GOOGLE_PROJECT_ID}/${IMAGE}" >> $BASH_ENV
             echo "export PR=$( [[ -z ${CIRCLE_PULL_REQUEST+x} ]] && echo false || echo $(basename ${CIRCLE_PULL_REQUEST}) )" >>$BASH_ENV
             export PR=$( [[ -z ${CIRCLE_PULL_REQUEST+x} ]] && echo false || echo $(basename ${CIRCLE_PULL_REQUEST}) )
-            echo "export HELM_TEST_NAME=pr-${PR}" >> $BASH_ENV
             sudo service apport stop
             curl https://baltocdn.com/helm/signing.asc | sudo apt-key add -
             sudo apt-get install apt-transport-https --yes
```

Comment:
Don't need these tools anymore. These are just for the helm part.

Reply:
Refactored.

Tools Installation was removed.